docs/markdown/snippets/pkg_config_enhancements.md

```diff
@@ -0,0 +1,4 @@
+## Enhancements to the pkg_config_path argument
+
+Setting sys_root in the [properties] section of your cross file will now set
+PKG_CONFIG_SYSROOT_DIR automatically.
```

Comment:
Should specify that this will only be set for dependencies that aren't set as `native: true`.